From ecf15acff28361b8f194f5232204792d915474fc Mon Sep 17 00:00:00 2001
From: Arthur Case <arthur@studio-automatique.com>
Date: Tue, 9 Jun 2015 17:07:13 +0800
Subject: [PATCH] Add required fields attributes

Some dashboard.promotions views were missing a fields attribute. For Django 1.8 compatibilty.
---
 src/oscar/apps/dashboard/promotions/views.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/oscar/apps/dashboard/promotions/views.py b/src/oscar/apps/dashboard/promotions/views.py
index a9bc2f1..45005b0 100644
--- a/src/oscar/apps/dashboard/promotions/views.py
+++ b/src/oscar/apps/dashboard/promotions/views.py
@@ -174,6 +174,8 @@ class CreateMultiImageView(CreateView):
 
 class CreateAutomaticProductListView(CreateView):
     model = AutomaticProductList
+    fields = ['name', 'description', 'link_url', 'link_text', 'method',
+              'num_products']
 
 
 class CreateHandPickedProductListView(CreateView):
@@ -279,14 +281,18 @@ class UpdateSingleProductView(UpdateView):
 
 class UpdateImageView(UpdateView):
     model = Image
+    fields = ['name', 'link_url', 'image']
 
 
 class UpdateMultiImageView(UpdateView):
     model = MultiImage
+    fields = ['name', 'images']
 
 
 class UpdateAutomaticProductListView(UpdateView):
     model = AutomaticProductList
+    fields = ['name', 'description', 'link_url', 'link_text', 'method',
+              'num_products']
 
 
 class UpdateHandPickedProductListView(UpdateView):
-- 
2.1.0

